app/Jobs/SendReplyToCustomer.php

```diff
@@ -223,6 +226,27 @@ public function handle()
             }
         }
 
+        // TODO copy reply to sent only if configured
+        $client = \MailHelper::getMailboxClient($mailbox);
+        $client->connect();
+
+        // TODO this needs fixing to be properly encoded and all
+        $content = "From: " . $mailbox->getMailFrom()['address'] . "\r\n"
```

Comment:
I am currently stuck here, since I am not a php/laravel developer. What is the correct way to construct a valid and properly escaped mail string in php?